roles/ansible_icinga_business_process/tasks/configuration_checks_business_process.yml

```diff
@@ -1,19 +1,21 @@
 ---
 
-- assert:
+- name: check the configuration of the business process
+  assert:
     that:
       - businessprocess.title is defined
```

Comment:
You probably have to check for non-emptiness, too. e.g.: `businessprocess.title != ""`